gatsby-config.js

```diff
@@ -6,6 +10,7 @@ module.exports = {
   },
   plugins: [
     `gatsby-plugin-react-helmet`,
+    `@contentful/gatsby-transformer-contentful-richtext`,
```

Comment:
so we need this? it's deprecated. 
https://github.com/contentful/gatsby-transformer-contentful-richtext

Reply:
we need this for serving rich text that includes HTML. Not able to find a suitable replacement yet.